docs/index.rst

```diff
@@ -20,6 +20,23 @@ Pangeo
 A community platform for Big Data geoscience
 ============================================
 
+What is Pangeo?
+^^^^^^^^^^^^^^^
+
+Pangeo is first and foremost a community promoting open, reproducible, and scalable science.
+This community provides documentation, develops and maintains software, and deploys
+computing infrastructure to make scientific research and programming easier.
+The Pangeo software ecosystem involves
+open source tools such as xarray, iris, dask, jupyter, and many other packages.
+There is no single software package called "pangeo"; rather, the Pangeo project serves
+as a coordination point between scientists, software, and computing infrastructure.
+On this website, scientists can find guides for accessing data and performing analysis
+using these tools. 
+Developers and IT professionals can find
```

Comment:
I wonder if limiting this to only developers and IT professionals is a bit exclusive? I'm okay either way, but this might come across as slightly daunting. Obviously can't please everyone.

Reply:
That was sort of my intention with using that language. Normal "science" users are less likely to be deploying software like jupyter hub on a cluster or cloud-based service. If this is something they need to do or that Pangeo has now made easy to do/understand for non-IT professionals then the instructions are there to view. I thought saying something like "IT professionals" gave the idea that there are very technical things that may be involved in deploying these types of technologies. This was only meant for these specific complex topics, if this sentence comes off as "you can't work with Pangeo instructions unless you have a CS degree" then yes we should change it.

Reply:
Maybe the sentence could be turned around: 
On this website, users interested in joining the community can find guides for accessing data and performing analysis using these tools (i.e. Geoscience Use Cases), together with detailed instructions for deploying cluster or cloud-based infrastructures.